include/solvers/navier_stokes_base.h

```diff
@@ -324,6 +324,9 @@ class NavierStokesBase : public PhysicsSolver<VectorType>
   TableHandler                                 kinetic_energy_table;
   AverageVelocities<dim, VectorType, DofsType> average_velocities;
 
+  // Tolerance to compare doubles
+  double epsilon = 1e-6;
+
```

Comment:
No need to define that as a class member. You can define it locally where you need it. It is not a critical parameter.